app/gui/src/dashboard/hooks/backendHooks.ts

```diff
@@ -595,11 +597,15 @@ export function useNewProject(backend: Backend, category: Category) {
           throw error
         })
         .then((createdProject) => {
+          const parent = getAsset(parentId)
```

Comment:
I think parent may not be in assets?

Reply:
theoretically yeah :/
not sure if there's a good solution though
i think ideal solution would be if `createdProject` contained `ensoPath`